src/main/java/org/generationcp/middleware/manager/ontology/TermDataManagerImpl.java

```diff
@@ -38,6 +39,11 @@ public Term getTermById(final Integer termId) throws MiddlewareException {
 		return Term.fromCVTerm(daoFactory.getCvTermDao().getById(termId));
 	}
 
+	@Override
+	public List<Term> getTermByIds(final List<Integer> termIds) throws MiddlewareException {
```

Comment:
Can we use `OntologyDataManager#getTermsByIds` instead of `TermDataManager` in `TermValidator` in BMSAPI?